core/src/test/scala/integration/kafka/network/DynamicConnectionQuotaTest.scala

```diff
@@ -58,11 +61,13 @@ class DynamicConnectionQuotaTest extends BaseRequestTest {
   @BeforeEach
   override def setUp(testInfo: TestInfo): Unit = {
     super.setUp(testInfo)
-    TestUtils.createTopic(zkClient, topic, brokerCount, brokerCount, servers)
+    admin = TestUtils.createAdminClient(brokers, listener)
```

Comment:
This can be simplified with:
```
admin = createAdminClient(listener)
```

Reply:
Thanks for the suggestion. I've changed it.